my-sms/spec/controllers/students_controller_spec.rb

```diff
@@ -134,8 +134,10 @@
       delete :destroy, { id: student.to_param }
     end
 
+    before { student }
```

Comment:
Small note, this could also be `let!(:student) { create(:student) }`, where the `let!` ensures this block is evaluated before your test. However, as you've got `let(:student) { create(:student) }` outside all the context blocks at the top of this test, I personally think this is a neater solution that's just as readable.